bilby/gw/waveform_generator.py

```diff
@@ -75,29 +76,21 @@ def __init__(self, duration=None, sampling_frequency=None, start_time=0, frequen
         if isinstance(parameters, dict):
             self.parameters = parameters
         self._cache = dict(parameters=None, waveform=None, model=None)
-        utils.logger.info(
-            "Waveform generator initiated with\n"
-            "  frequency_domain_source_model: {}\n"
-            "  time_domain_source_model: {}\n"
-            "  parameter_conversion: {}"
-            .format(utils.get_function_path(self.frequency_domain_source_model),
-                    utils.get_function_path(self.time_domain_source_model),
-                    utils.get_function_path(self.parameter_conversion))
-        )
+        utils.logger.info(f"Waveform generator instantiated: {self}")
```

Comment:
`logger` is imported but `utils.logger` is used instead, I think this should be consistent.